### PR TITLE
Create ge-Ge ( A german translator)

### PR DESCRIPTION
**Pull request by Mayshank18**
A translation in german-GERMANY code.

### CODE REVIEW

File: assets/translations/ge-GE.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "Create_new_account": "Neuen Account erstellen",
+  "Actions_History": "Aktionsverlauf",
+  "Empty_history": "Leere Geschichte",
+  "History_Settings_Page": "Seite Verlaufseinstellungen",
+  "Lamps_Page": "Lampen Seite",
+  "Name:_": "der Name: {}",
+  "Room:_": "Zimmer: {}",
+  "Lamps_Settings_Page": "Seite Lampeneinstellungen",
+  "Welcome_to_your_Smart_Home": "Willkommen in Ihrem Smart Home",
+  "Scene_Settings_Page": "Seite "Szeneneinstellungen"",
```
Comment (guyluz11) [inline assets/translations/ge-GE.json:11]: You have quotation marks inside quotation marks.
Please removed the inside couple

**Comment by Mayshank18** [comment]
I did the required changes you asked for, please review !!

**Comment by guyluz11** [comment]
Thanks for the pr @Mayshank18

**Comment by guyluz11** [comment]
If you would like to help us make the world a better place a ⭐star🌟 to the project will help.

Happy hacktoberfest :)

**Pull request merged**